src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeGCSClient.java

```diff
@@ -296,7 +296,8 @@ public void download(
           }
 
           logger.debug("Starting download without presigned URL", false);
-          blob.downloadTo(localFile.toPath());
+          blob.downloadTo(
```

Comment:
Is there test coverage for this? Or are there any existing tests that this would affect?

Reply:
We change the way we access GCS. So existing tests will do.

Reply:
So this is not a behavior change? Does JDBC do the decompression somewhere else previously?

Reply:
No behavior change. In the same method below it does decompression. So download file as is and then decompress locally.Same logic as pre-signed url.